client/my-sites/importer/newsletter/importer.scss

```diff
@@ -41,6 +41,12 @@ $newsletter_importer_line-height: 20px;
 		padding: 0;
 		margin: 0;
 		box-shadow: none;
+
+		.calypso-notice.is-error {
+			a, .button.is-link {
+				text-decoration: underline;
+			}
+		}
```

Comment:
I'm adding back the underline for this specific case (error notice in the importer card, in the newsletter importer). We may get different design direction here: p1749593376510159-slack-C03NLNTPZ2T In that case we can replace/remove the rule.